docs/release_notes/v1.11.5.md

```diff
@@ -19,3 +49,58 @@ An issue was detected in the internal implementation of service invocation when
 ### Solution
 
 We fixed the issue in Dapr 1.12.0 and 1.11.5.
+
+## Fixed: Fix invoke remote stream from closing stream before EOF
+
+### Problem
+
+If an invoke stream is using resiliency, then service invocation will return with an EOF error rather than successfully returning the result.
+
+### Impact
+
+When `ServiceInvocationStreaming` is enabled, service invocation is unusable when using resiliency.
+
+### Root cause
+
+A bug in the code meant that the gRPC stream was closed before the EOF was sent to the client.
+
+### Solution
+
+Fix the code so that the EOF is sent before the stream is closed.
+
+## Fixed: Downgrading from 1.12 to 1.11
+
+### Problem
+
+When downgrading from 1.12 to 1.11, the Dapr sidecar injector will fail to inject the sidecar into pods, printing CA verification errors in the logs.
+
+### Impact
+
+Users who downgrade the Dapr installation from 1.12 to 1.11 will result in their Dapr cluster being in broken state.
+
+### Root cause
+
+1.12 made a change that the Injector's serving certificate is now signed by the Dapr CA, rather than a separate CA managed through Helm.
```

Comment:
```suggestion
1.12 made a change that the Injector's serving certificate is now signed by the Dapr CA (Certification Authority), rather than a separate CA managed through Helm.
```